src/website/index.css

```diff
@@ -6,3 +6,7 @@ body {
 main {
   display: block;
 }
+
+* {
```

Comment:
This isn't necessary. The app's root: https://github.com/mineral-ui/mineral-ui/blob/7d6f2ab63196f8819fe745b853bc8d3c2135157a/src/website/app/App.js#L69-L71

includes the style reset which sets this property and all of its children to inherit it: https://github.com/mineral-ui/mineral-ui/blob/7d6f2ab63196f8819fe745b853bc8d3c2135157a/src/styles/componentStyleReset.js#L20-L28

Reply:
addressed